### DISCUSSION TITLE
I need your knowledge

### DISCUSSION THREAD

**Discussion by matheine** [elrumo/macOS_Big_Sur_icons_replacements#1134] [General]
I want to make my own app Icons for macOS Big Sur, but every Icon I made is in a different size to all the app icons designs by apple. What size do I need?

**Comment by matheine** [comment]
![screenshot](https://user-images.githubusercontent.com/68849310/113573354-42dc9300-961a-11eb-9780-780ee7151ce7.png)

@gl-Gho 
left: Apple
right: mine

**Comment by ghost** [comment]
Download the template from https://macosicons.com/resources and follow these tutorials: https://www.youtube.com/playlist?list=PLR1JA1_plMXoauTdVuRGOrXZFT4o9e0ub

**Comment by V-R-Technology** [comment]
So you're asking for the corner radius. canvas size: 1000x1000. icon size: 824x824. round the corners about 20-25% (I recommend 23 or 24%). its OK if not exact, but you won't notice the difference.
